pkg/relayer/proxy/synchronous.go

```diff
@@ -58,7 +58,9 @@ type synchronousRPCServer struct {
 // relay requests and forwards them to the supported proxied service endpoint.
 // It takes the serviceId, endpointUrl, and the main RelayerProxy as arguments
 // and returns a RelayServer that listens to incoming RelayRequests.
-// TODO_BLOCKER(red0ne): Use TCP server for the Gateway->RelayMiner communication.
+// TODO_RESEARCH: Currently, the communication between the AppGateServer and the
```

Comment:
OPTIONAL: consider creating a ticket and linking all the `TODO_RESERACH` comments in this PR to that ticket: `TODO_RESEARCH(#xxx): ...`.

1. You can link to this PR in the new issue
2. You can upload the screenshot for reference

![Screenshot 2024-06-04 at 2 07 03 PM](https://github.com/pokt-network/poktroll/assets/1892194/8729a2a6-a52e-4623-9443-5fed959a8e35)